conf/locale/ru/LC_MESSAGES/djangojs.po

```diff
@@ -11944,3 +11944,12 @@ msgstr ""
 #: cms/templates/js/video/transcripts/messages/transcripts-use-existing.underscore
 msgid "Use Current Transcript"
 msgstr "Использовать субтитры"
+
+msgid "Cohort Management"
+msgstr "Руководитель когорты"
```

Comment:
Are these translations have right context? I see only `.po` file and in this case I can say that these translations are wrong.